core/display.js

```diff
@@ -84,790 +85,785 @@ export default function Display(defaults) {
     }
 
     if (this._prefer_js === null) {
-        Util.Info("Prefering javascript operations");
+        Log.Info("Prefering javascript operations");
         this._prefer_js = true;
     }
 
     // Determine browser support for setting the cursor via data URI scheme
     if (this._cursor_uri || this._cursor_uri === null ||
             this._cursor_uri === undefined) {
-        this._cursor_uri = Util.browserSupportsCursorURIs();
+        this._cursor_uri = cursorURIsSupported();
     }
 
-    Util.Debug("<< Display.constructor");
+    Log.Debug("<< Display.constructor");
 };
 
-(function () {
-    "use strict";
```

Comment:
Modules are strict-by-default, according to the spec (http://www.ecma-international.org/ecma-262/6.0/#sec-strict-mode-code <-- see the second bullet)